circle.yml

```diff
@@ -0,0 +1,19 @@
+test:
+  override:
+    - scripts/run-circle.sh
+
+machine:
+  # Courtesy of https://pewpewthespells.com/blog/building_python_on_circleci.html
+  pre:
+    - export PATH=/usr/local/bin:$PATH:/Users/distiller/Library/Python/2.7/bin
+    - pip install --user --ignore-installed --upgrade virtualenv
+    - ln -s $HOME/Library/Python/2.7/bin/virtualenv /usr/local/bin/virtualenv
+    - cd "$(brew --repository)" && git fetch && git reset --hard origin/master
+    - brew update
+
+dependencies:
+  override:
+    - make install-core
+
+  cache_directories:
+    - ~/.cache/hypothesis-build-runtimes
```

Comment:
A couple of reasons.

* I had an extremely hard time persuading Circle to actually set an environment variable which pointed to a path in the home directory. As a result we're redefining where .runtimes lives using an environment variable in travis but using the default in Circle.
* Circle only caches stuff that happens in the dependency step, so most of those directories aren't populated by the time it stores its cache.
* I suspect some of those directories are dead and don't matter in Travis either.